build.rs

```diff
@@ -52,7 +53,12 @@ fn main() {
             .map(|m| format!("src/prost/{}.rs", m))
             .collect::<Vec<_>>(),
         "src/prost",
-        GenOpt::All,
+        GenOpt::MUT
+            | GenOpt::TRIVIAL_SET
+            | GenOpt::HAS
+            | GenOpt::TAKE
+            | GenOpt::CLEAR
+            | GenOpt::MESSAGE,
```

Comment:
Nicer to start with `All` and then remove trivial get?

Reply:
I recommend removing the remaining parts one by one instead of adding the eliminated parts one by one.